traitlets/traitlets.py

```diff
@@ -2386,296 +2362,311 @@ def info_rst(self):
         return self._info(as_rst=True)
 
 
-class Container(Instance):
-    """An instance of a container (list, set, etc.)
-
-    To be subclassed by overriding klass.
-    """
-    klass = None
-    _cast_types = ()
-    _valid_defaults = SequenceTypes
-    _trait = None
-
-    def __init__(self, trait=None, default_value=None, **kwargs):
-        """Create a container trait type from a list, set, or tuple.
-
-        The default value is created by doing ``List(default_value)``,
-        which creates a copy of the ``default_value``.
+class Mutable(Instance):
 
-        ``trait`` can be specified, which restricts the type of elements
-        in the container to that TraitType.
+    def default(self, obj=None):
```

Comment:
Mutable's not doing anything here. I think we can get rid of it to cutdown on the significant increase in inheritance complexity in this PR.